AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/SubscriptionSync/Support/MockModelReconciliatinQueue.swift

```diff
@@ -15,7 +15,8 @@ import Combine
 class MockModelReconciliationQueue: ModelReconciliationQueue {
 
     public static var mockModelReconciliationQueues: [String: MockModelReconciliationQueue] = [:]
-
+    internal var count: Int
+    internal var isFullSync: Bool
```

Comment:
I think these are remnants from previous implementation, remove if no longer needed

Reply:
True. I forgot about this. Removed